server.js

```diff
@@ -15,9 +15,102 @@ const welcomeMessage = {
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
 const messages = [welcomeMessage];
+let maxID = Math.max(...messages.map((c) => c.id));
+
+//validation function
+function validateInputData(name, text, res) {
```

Comment:
Good. It is better not to pass res into a function, as it makes it hard to test. Better to return true/false and allow the request handler function to return a 400 response if needed